### PR TITLE
NO-TICKET - Testing new Swap

### PR DESCRIPTION
**Pull request by gslevinkov**

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt [hunk 1]
```diff
@@ -121,6 +122,15 @@ object NetworkModule : InjectionModule {
                 interceptor = null
             )
         }
+
+        single(named(SwapModule.JUPITER_RETROFIT_QUALIFIER)) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt:126]: @NikolaiEmelianov there is a tradition to declare retrofit instances in NetworkModule only fyi

File: app/src/main/java/org/p2p/wallet/swap/jupiter/repository/transaction/JupiterSwapTransactionMapper.kt [hunk 2]
```diff
@@ -0,0 +1,81 @@
+package org.p2p.wallet.swap.jupiter.repository.transaction
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/jupiter/repository/transaction/JupiterSwapTransactionMapper.kt:1]: just moved them into packages, nothing new

**Review by NikolaiEmelianov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/sdk/facade/RelaySdkFacade.kt [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 import org.p2p.solanaj.utils.crypto.encodeToBase58
 import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
 import org.p2p.wallet.sdk.RelaySdk
 import org.p2p.wallet.sdk.facade.mapper.SdkMethodResultMapper
 import org.p2p.wallet.utils.Base58String
-import org.p2p.wallet.utils.toBase58Instance
 import kotlinx.coroutines.withContext
 
-class SdkSignedTransaction(val transaction: Base58String)
+class SdkSignedTransaction(
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/sdk/facade/RelaySdkFacade.kt:13]: if it's network model, please add Response to class name

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/sdk/facade/RelaySdkFacade.kt]: fixed! thanks

**Pull request merged**